docs/web-analytics.md

```diff
@@ -4,12 +4,12 @@ title: Add web analytics to your site
 sidebar_label: Web Analytics
 ---
 
-The purpose of this file is to document how you can enable a Web Analytics solution. 
-Currently Richie only supports by default the Google Analytics using the Google Tag Manager Javascript.
-But it is possible with little cost to add support for other web analytics solutions.
+Richie has native support to [Google Analytics](#google-analytics) and [Google Tag Manager](#google-tag-manager) Web Analytics solutions.
+The purpose of this file is to document how you can enable one of the supported Web Analytics providers
+and how you can extend Richie with an alternative solution.
 
 ## Google Analytics
-Next, it is decribed how you can configure the **Google Analytics** on your Richie site.
+Next, it is described how you can configure the **Google Analytics** on your Richie site.
 
 - Add the `WEB_ANALYTICS_ID` setting, with your Google Analytics tracking id code.
```

Comment:
It miss one intruction right?
```suggestion
- Add the `WEB_ANALYTICS_ID` setting, with your Google Analytics tracking id code.
- Add the `WEB_ANALYTICS_PROVIDER` setting with the `google_analytics` value.
```

Reply:
No, the `google_analytics` is the default provider.